base/Base.jl

```diff
@@ -241,6 +241,7 @@ include("filesystem.jl")
 using .Filesystem
 include("cmd.jl")
 include("process.jl")
+include("ttyhascolor.jl")
```

Comment:
Does this need to be its own file? Otherwise it seems like it makes sense to keep it with the implementation of TTY.

Reply:
I think ,we can shift this to stdlib/REPL/src/Terminal.jl

Reply:
@JeffBezanson The code in ttyhascolor.jl needs to come after process.jl due to the use of `success`. stream.jl is included before process.jl, and we were getting compilation errors because of this.

Reply:
Ok, I figured it was something like that.